src/test/java/org/jboss/xavier/integrations/migrationanalytics/business/VMWorkloadInventoryCalculatorTest.java

```diff
@@ -182,4 +182,21 @@ public void calculate_jsonV1_0_0_GivenWithMissingAttributes_ShouldReturnCalculat
         assertThat(modelList.stream().filter(e -> e.getVmName().equalsIgnoreCase("oracle_db"))
                 .findFirst().get().getCluster()).isNull();
     }
+
+    @Test
+    public void calculate_jsonV1_0_0_GivenWithCoresEquals0_ShouldReturn0CpuCores() throws IOException, ParseException {
```

Comment:
`ParseException` never thrown
```suggestion
    public void calculate_jsonV1_0_0_GivenWithCoresEquals0_ShouldReturn0CpuCores() throws IOException {
```

Reply:
+1